compiler/rustc_incremental/src/persist/load.rs

```diff
@@ -209,10 +209,7 @@ pub fn load_query_result_cache<'a>(
     sess: &'a Session,
     definitions: &Definitions,
 ) -> Option<OnDiskCache<'a>> {
-    if sess.opts.incremental.is_none() {
-        return None;
-    }
-
+    sess.opts.incremental.as_ref()?;
```

Comment:
I'd leave this part unchanged.
To me `?` conveys a meaning of  "I want to use this value but the container is in the way". Here the intent is about looking at the container itself, so the more explicit code makes it easier to understand IMO.

Reply:
dropped